docs/moderator-handbook.md

```diff
@@ -468,3 +463,47 @@ This is a standard message notifying you that we've reviewed your pull request a
 
 Thank you and happy coding.
 ```
+
+> When PR adds links to external resources.
+
+```markdown
+Thank you for your pull request.
+
+I am closing this pull request. Please add links and other details to the challenge's corresponding guide article instead.
+
+If you think I'm wrong in closing this issue, please reopen it and add further clarification. Thank you, and happy coding.
+```
+
+## Closing Invalid Issues
+
+> When an issue relates to the camper's code.
+
+```markdown
+Thank you for reporting this issue.
+
+This is a standard message notifying you that this issue seems to be a request for help. Instead of asking for help here, please click the **"Help"** button on the challenge on freeCodeCamp, which will help you create a question in the right part of the forum. Volunteers on the forum usually respond to questions within a few hours and can help determine if there is an issue with your code or the challenge's tests.
+
+If the forum members determine there is nothing wrong with your code, you can request this issue to be reopened.
+
+Thank you and happy coding.
+```
+
+> When an issue is duplicate of an earlier issue
+
+```markdown
+Thank you for reporting this issue.
+
+This is a standard message notifying you that this issue appears to be very similar to issue #XXXXX, so I am closing it as a duplicate.
+
+If you think I'm wrong in closing this issue, please reopen it and add further clarification. Thank you and happy coding.
+```
+
+> When an issue is fixed in staging.
+
+```markdown
+Thank you for reporting this issue.
+
+This is a standard message notifying you that the problem you mentioned here is present in production, but that it has already been fixed in staging. This means that the next time we push our staging branch to production, this problem should be fixed. Because of this, I'm closing this issue.
```

Comment:
```suggestion
This is a standard message notifying you that the problem you mentioned here is present in production, but that it has already been fixed in staging. This means that the next time we push our staging branch to production, this problem should be fixed. Because of this, we're closing this issue.
```